Example/Example/Models/PlaceholderUser.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Example.Models
```

Comment:
is this actually used?

Reply:
Yes, it's object representation of JSON used in HTTP example.